test/e2e/framework/nodes_util.go

```diff
@@ -63,7 +63,7 @@ func etcdUpgradeGCE(target_storage, target_version string) error {
 		os.Environ(),
 		"TEST_ETCD_VERSION="+target_version,
 		"STORAGE_BACKEND="+target_storage,
-		"TEST_ETCD_IMAGE=3.2.18-0")
+		"TEST_ETCD_IMAGE=3.2.24-0")
```

Comment:
Please also modify test/kubemark/start-kubemark.sh

Reply:
Done, plus I found one other test that needed an update too.